Draft/draft-ietf-rtcweb-stun-consent-freshness.xml

```diff
@@ -167,12 +167,14 @@
       document. The consent mechanism does not update the ICE procedures
       defined in <xref target="RFC5245"></xref>.</t>
 
-      <t>Consent is obtained only by full ICE implementations. An ICE-lite
-      implementation as defined in section 2.7 of <xref
-      target="RFC5245"></xref> will not generate consent checks, but will just
-      respond to consent checks it receives. No changes are required to
-      ICE-lite implementations in order to respond to consent checks, as they
-      are processed as normal ICE connectivity checks.</t>
+      <t>Consent is obtained only by full ICE implementations. ICE-lite agent
+      defined in section 2.7 of <xref target="RFC5245"></xref> does not
+      generate connectivity checks or runs the ICE state machine, it only
+      needs to be able to respond to connectivity checks. Hence an ICE-lite
+      agent will not generate consent checks, but will just respond to consent
+      checks it receives. No changes are required to ICE-lite implementations
```

Comment:
Grammar here is poor.

> [...] An ICE-lite agent (as defined in Section 2.7 of <xref target="RFC5245"/>) does not generate connectivity checks or run the ICE state machine.  An ICE-lite agent does not generate consent checks, it will only respond to any checks that it receives. [...]